client/test/load-suite.js

```diff
@@ -0,0 +1,15 @@
+function requireTestFiles( config, path = '' ) {
+	Object.keys( config ).map( ( folderName ) => {
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/commit/060e26f5c12fb9d71d95ce48811dba503a37707a.